### PR TITLE
GAX rename: RpcFuture -> ApiFuture

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**

### CODE REVIEW

File: google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/deprecated/Message.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ public PubsubMessage apply(Message message) {
     }
 
     @Override
-    protected ByteString getByteString() {
+    public ByteString getByteString() {
```
Comment (vam-google) [inline google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/deprecated/Message.java]: Is it related to the RpcFuture -> ApiFuture rename? I is not related to the change, but it seems strange that the class Message under the "deprecated" directory is not declared as @Deprecated .

Comment (garrettjonesgoogle) [inline google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/deprecated/Message.java]: Oh this was a mistake, I didn't mean to include this change.

**Comment by garrettjonesgoogle** [comment]
Feedback addressed, PTAL

**Comment by garrettjonesgoogle** [comment]
As for the codacy-bot comments, it is wrong in this case. We specifically want println statements because it is just example code.

**Pull request merged**